sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/_package/model_package.py

```diff
@@ -202,17 +203,17 @@ class ModelPackage(Resource, PackageRequest):
     def __init__(
         self,
         *,
-        name: Optional[str] = None,
+        name: str,
+        inferencing_server: Union[AzureMLOnlineInferencingServer, AzureMLBatchInferencingServer],
```

Comment:
Do we need to add: `TritonInferencingServer` and `CustomInferencingServer` too?

Reply:
Not supported yet, once they are we will add...